data/net/minecraft/world/entity/Entity.mapping

```diff
@@ -284,6 +310,21 @@ CLASS net/minecraft/world/entity/Entity
 	METHOD killed (Lnet/minecraft/server/level/ServerLevel;Lnet/minecraft/world/entity/LivingEntity;)V
 		ARG 1 level
 		ARG 2 killedEntity
+	METHOD lambda$collide$1 (Lnet/minecraft/world/entity/Entity;)Z
+		ARG 0 unused
+	METHOD lambda$findDimensionEntryPoint$4 (Lnet/minecraft/server/level/ServerLevel;Lnet/minecraft/BlockUtil$FoundRectangle;)Lnet/minecraft/world/level/portal/PortalInfo;
+		ARG 2 exitPortal
+	METHOD lambda$getDisplayName$9 (Lnet/minecraft/network/chat/Style;)Lnet/minecraft/network/chat/Style;
+		ARG 1 currentStyle
+	METHOD lambda$isInWall$2 (Lnet/minecraft/world/level/block/state/BlockState;Lnet/minecraft/core/BlockPos;)Z
+		ARG 1 state
+		ARG 2 pos
+	METHOD lambda$move$0 (Lnet/minecraft/world/level/block/state/BlockState;)Z
+		ARG 0 loadedState
+	METHOD lambda$null$3 (Lnet/minecraft/world/level/block/state/BlockState;Lnet/minecraft/core/BlockPos;)Z
+		ARG 2 pos
```

Comment:
I guess it's lambda method in `Entity#isInWall`, one that takes BlockState and BlockPos as arguments. That is the only lambda function that takes BlockState and BlockPos in Entity class. Not sure why that happens

Reply:
Not sure, Weird thing is, There is only one lambda method that takes BlockState and BlockPos as arguments. and that is one in `Entity#isInWall`

Reply:
Found it, it's second lambda function in `Entity#findDimensionEntryPoint`, in `BlockUtil.getLargestRectangleAround`

Reply:
This has been marked resolved but doesn't seem to have changed. Could you try to remap it or if that does the same thing manually find the identifier?

Reply:
Never mind; looking at the byte code its identifier is indeed `lambda$null$4`. Seems that it calls it that because it's embedded in another lambda. My bad, I should've checked this before creating the comment.